CMakeLists.txt

```diff
@@ -474,6 +476,10 @@ if(QUDA_HETEROGENEOUS_ATOMIC AND QUDA_JITIFY)
   message(SEND_ERROR "QUDA_HETEROGENEOUS_ATOMIC=ON does not support JITIFY)")
 endif()
 
+if(QUDA_CLOVER_RECONSTRUCT AND NOT QUDA_CLOVER_DYNAMIC)
```

Comment:
thoughts on replacing this if by using
```
cmake_dependent_option(QUDA_CLOVER_RECONSTRUCT "set to ON to enable compressed clover storage" OFF "QUDA_CLOVER_DYNAMIC" OFF)
```
?
```

Reply:
No strong thoughts.  Personally
I don’t like options disappearing from `ccmake` when certain options are set, and I like to see them, even if they error when enabled / disabled.

Reply:
works for me. I was just annoyed that to disable dynamic I had to switch to advanced to disable reconstruct separately.